operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Memcached: enable providing additional extended options (`-o/--extended`) via `<cache-section>.extraExtendedOptions`. #5353
 * [ENHANCEMENT] Memcached exporter: enable adding additional CLI arguments via `memcachedExporter.extraArgs`. #5353
 * [ENHANCEMENT] Memcached: allow mounting additional volumes to the memcached and exporter containers via `<cache-section>.extraVolumes` and `<cache-section>.extraVolumeMounts`. #5353
+* [ENHANCEMENT] Adds the optional appProtocol field to the memberlist service. This allows memberlist to work with istio protocol selection. You can configure the appProtocol via the `mimir.memberlistAppProtocol` value. #5177
```

Comment:
Can you please move this to the `main / unreleased` section of the changelog? I think it got left in the 5.0.0 section after rebasing :)